README.md

```diff
@@ -32,9 +32,29 @@ func main() {
 
 ## Building
 
-You need Docker and then run
-
-    ./script/build
+You need Docker and ``make`` and then run the ``binary`` target. This
+will create binary for all platform in the `bundles` folder. 
+
+```bash
+λ make binary
```

Comment:
yeah, I saw the double "ticks" as well, but those don't show in the rendered page, so didn't want to bother you with that :-)

The prompt is something we try to be consistent with though.